build.sbt

```diff
@@ -224,7 +224,10 @@ lazy val common = crossProject(JSPlatform, JVMPlatform)
   .in(file("scalameta/common"))
   .settings(
     publishableSettings,
-    libraryDependencies += "com.lihaoyi" %% "sourcecode" % "0.1.3",
+    libraryDependencies ++= List(
+      "com.lihaoyi" %% "sourcecode" % "0.1.3",
+      "com.lihaoyi" %% "pprint" % "0.5.3"
+    ),
```

Comment:
We can make these dependencies even more ubiquitous by adding them to `langmeta`. I think that `semanticdb3` would be an overkill though.

Reply:
Done.